app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,73 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
```

Comment:
For this to be an MVC framework app, we want to load up our data in the controller. 

Do you want to find all the tweets `@tweets = Tweet.all`? Or just the tweets for the user that's logged in `@tweets= current_user.tweets`?